content/en/developers/marketplace.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Marketplace
+type: documentation
+---
+
+<div class="alert alert-info">Want to be a marketplace partner? Contact us at <a href="mailto:marketplace@datadog.com">marketplace@datadog.com</a> to let us know your idea!</div>
+
+## Before you begin
+
+Before you begin the release process, you need your marketplace application approved by Datadog, access to the marketplace repo, a built integration and you need to set up Datadogs Developer toolkit, [ddev][1].
+
+## Releasing your integration
+
+Making a release requires the maintainer to update two files in the same pull request (PR).
+
+1. Update the CHANGELOG.md file
+    This file can be automatically updated by ddev using the following command:
+    ```
+    ddev release changelog <INTEGRATION_NAME> <VERSION>
+    ```
+    The command lists all merged PRs since the last release and creates a changelog entry based on the pull request labels. For changelog types, we adhere to those defined by [Keep a Changelog][2].
```

Comment:
```suggestion
    The command lists all merged PRs since the last release and creates a changelog entry based on the pull request labels. For changelog types, adhere to those defined by [Keep a Changelog][2].
```